docker-compose.yml

```diff
@@ -5,15 +5,43 @@ services:
             - "/var/run/docker.sock:/var/run/docker.sock"
```

Comment:
Since the gateway doesn't interact with the Docker API, do we need the Docker sock mount?

Reply:
No I must have left it in this version by mistake. Thank you for load testing. We have one more hop now. Between provider and the gateway.